console/src/components/CompactAppList/CompactAppList.js

```diff
@@ -0,0 +1,35 @@
+import kind from '@enact/core/kind';
+import {Row} from '@enact/ui/Layout';
+import React from 'react';
+import AppIconCell from '../AppIconCell';
+
+const CompactAppList = kind({
+	name: 'CompactAppList',
+
+	styles: {
+		className: 'compact'
+	},
+
+	handlers: {
+		onTabChange: (ev, {onSelect}) => {
+			if ((ev.keyCode === 13 || ev.type === 'click') && ev.currentTarget.dataset.tabindex) {
+				onSelect({index: parseInt(ev.currentTarget.dataset.tabindex)});
+			}
+		}
+	},
+
+	render: ({onTabChange, ...rest}) => (
```

Comment:
`rest` should get applied to the outermost element of the render.

Reply:
i don't have anything to pass to the div.

Reply:
You don't right now, but others in the future may. We always handle the `rest`.